examples/sgp4Prop/fsw/eci_inc/sgp4Wrapper.h

```diff
@@ -14,6 +15,8 @@ typedef struct  {
     double r[3];
     double v[3];
     double t;
+    double epoch;
```

Comment:
I'm not sure we really need to add `epoch` to this structure when `tle.epoch` can be used instead (and in fact is actually what the code is using for the comparison).

Reply:
Removed that epoch and replaced it with tle.epoch.